GDJS/Runtime/howler-sound-manager/howler-sound-manager.js

```diff
@@ -409,7 +429,9 @@ gdjs.HowlerSoundManager.prototype.preloadAudio = function (
 
   var loadedCount = 0;
   var that = this;
-  function onLoad() {
+  function onLoad(_, error) {
+    if(error) console.error(error);
```

Comment:
That's a good idea, add a description like: `console.error("There was an error while loading an audio file:", error);`